src/features/events/components/EventContactCard.tsx

```diff
@@ -0,0 +1,129 @@
+import CloseIcon from '@mui/icons-material/Close';
+import FaceOutlinedIcon from '@mui/icons-material/FaceOutlined';
+import FaceRetouchingOffOutlinedIcon from '@mui/icons-material/FaceRetouchingOffOutlined';
```

Comment:
Instead of importing these from their respective single-icon modules, import them all from `@mui/icons-material`, like so:

```tsx
import { Close, FaceOutlined, FaceRetouchingOff } from '@mui/icons-material`;
```